documentation/API-design-guidelines.md

```diff
@@ -1433,29 +1486,32 @@ _Termination rules regarding subscriptionExpireTime & subscriptionMaxEvents usag
 
 
 ##### Resource-based (explicit) example
-In this example, we illustrate a request for a device roaming status event subscription. Requester did not provide an anticipated expiration time for the subscription. In the response, server accepts this request and sets an event subscription end one year later. This is an illustration and each implementation is free to provide - or not - a subscription planned expiration date.
+In this example, we illustrate a request for a device roaming status event subscription. Requester did not provide an anticipated expiration time for the subscription but requested to get an event if the device is in roaming situation at subscription time and set the max limit to event to 10. In the response, server accepts this request and sets an event subscription end one year later. This is an illustration and each implementation is free to provide - or not - a subscription planned expiration date.
 
 Request:
 
 ```bash
 curl -X 'POST' \
-  'http://localhost:9091//device-status/v0/subscriptions' \
+  'http://localhost:9091/device-roaming-subscriptions/v1/subscriptions' \
   -H 'accept: application/json' \
   -H 'Content-Type: application/json' \
   -d
  ```
  ```json 
 {
-  "webhook": {
-    "notificationUrl": "https://application-server.com",
-    "notificationAuthToken": "c8974e592c2fa383d4a3960714"
-    }
-  "subscriptionDetail": {
-    "ueId": {,
-      "ipv4Addr": "192.168.0.1"
+  "protocol" : "HTTP",
+  "sink": "https://application-server.com",
+  "types" : [
+      "org.camaraproject.device-roaming-subscriptions.v1.roaming-status"
+  ],
+  "config": {
+    "subscriptionDetail": {
+      "device": {
+        "phoneNumber": "12345678"
```

Comment:
```suggestion
        "phoneNumber": "+346661113334"
```

Reply:
Actually, I am not too happy with "phoneNumber" because this is personal-identifiable information and the access_token was created after some user authentication and consent collection, and when the authorization server created the access token (and/or id token) the subject `sub` is known. 
There might be no need for subscriptionDetails because the subject is already known and "integrated" into the access token.
Only if this endpoint allows client credentials the details are needed. "client credentials" should be the exception in Camara.